drivers/bluetooth/hci/Kconfig

```diff
@@ -30,6 +30,12 @@ config BT_H5
 	  Bluetooth three-wire (H:5) UART driver. Implementation of HCI
 	  Three-Wire UART Transport Layer.
 
+config BT_RPMSG
+	bool "RPMsg HCI"
```

Comment:
Make it "HCI using RPMsg"? Or "HCI over RPMsg transport"?

Reply:
fixed